tslint-warnings.csv

```diff
@@ -25,9 +27,11 @@ file-name-casing,Enforces a consistent file naming convention,TSLINTGCPLQ3,tslin
 CWE 710 - Coding Standards Violation"
 forin,Requires a `for ... in` statement to be filtered with an `if` statement.,TSLINTTBFHNF,tslint,Non-SDL,Warning,Important,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,"398, 710","CWE 398 - Indicator of Poor Code Quality
 CWE 710 - Coding Standards Violation"
+function-constructor,Prevents using the built-in Function constructor.         ,TSLINT7Q9QUS,tslint,Non-SDL,Warning,Moderate,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,398,"CWE 398 - Indicator of Poor Code Quality"
```

Comment:
Looks like local build needed to update there descriptions after metadata update.
CWE for `function-constructor` now was updated in metadata, but not here.